src/contracts/CrossChainForwarder.sol

```diff
@@ -433,6 +513,21 @@ contract CrossChainForwarder is OwnableWithGuardian, ICrossChainForwarder {
     }
   }
 
+  /**
+  * @notice method to update the required confirmations of a receiver chain
+  * @param requiredConfirmationsByReceiverChain array of objects containing the requiredConfirmations for a specified
+           receiver chain id
+  */
+  function _updateRequiredConfirmationsForReceiverChain(
```

Comment:
Feels there should be a validation for confirmation being less than the count of bridge adapters, and more than 0

Reply:
Agree on the > 0, as we are setting 0 to not configured. But not sure I agree on the <= bridge adapters, as this would introduce the need for ordering on setting configurations, and it does not necessarily break the system, as we have the condition that if there are >= use all adapters. (We can add natspec describing this case)